e2e/wdio/core/tests/breadcrumb.e2e-spec.ts

```diff
@@ -0,0 +1,63 @@
+import { BreadcrumbPo } from '../pages/breadcrumb.po';
+import {
+    checkElementScreenshot,
+    getElementArrayLength,
+    isElementClickable,
+    mouseHoverElement,
+    refreshPage, saveElementScreenshot,
+    scrollIntoView, waitForPresent
+} from '../../driver/wdio';
+
+import {
+    linksExample, linksHoverState
+} from '../fixtures/testData/breadcrumb.tags';
+
+
+describe('Breadcrumb test suite:', function() {
+
+    const breadcrumbPage: BreadcrumbPo = new BreadcrumbPo();
+    const { links, disabledLinks } = breadcrumbPage;
+
+    beforeAll(() => {
+        breadcrumbPage.open();
+    }, 1);
+
+    afterEach(() => {
+        refreshPage();
+        waitForPresent(links);
+    }, 1);
+
+    it('should check links', () => {
+        const linksLength = getElementArrayLength(links);
+        for (let i = 0; i < linksLength; i++) {
+            scrollIntoView(links, i);
+            expect(isElementClickable(links, i)).toBe(true);
+            checkLinkHoverState(links, linksExample + linksHoverState + '-' + i, 'breadcrumb-links', i);
+        }
+    });
+
+    it('should check disable links', () => {
+        const disabledLinksLength = getElementArrayLength(disabledLinks);
+        for (let i = 0; i < disabledLinksLength; i++) {
+            scrollIntoView(disabledLinks, i);
+            expect(isElementClickable(disabledLinks, i)).toBe(false);
+        }
+    });
+
+    describe('Check visual regression', function() {
```

Comment:
still missing focus and active state checks

Reply:
I cannot add focus state due to this issue https://github.com/SAP/fundamental-ngx/issues/4840. Will add active state checks